webhooks-extension/endpoints/credentials_test.go

```diff
@@ -0,0 +1,243 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+		http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package endpoints
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"reflect"
+	"strings"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestCreateBadAccessToken(t *testing.T) {
+	r := dummyResource()
+	namespace := "default"
+	r.K8sClient.CoreV1().Namespaces().Create(&corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}})
+	badAccessToken := credential{
+		Name: "badToken",
+	}
+	expectedError := fmt.Sprintf("Error: AccessToken must be supplied")
+	createAndCheckCredential(namespace, badAccessToken, expectedError, r, t)
+
+	// Verify no credentials have been created
+	checkCredentials(namespace, []credential{}, "", r, t)
+}
+
+func TestAccessTokenWithSecret(t *testing.T) {
+	r := dummyResource()
+	namespace := "default"
```

Comment:
I would suggest adding in another test similar but with non default namespace to ensure namespaces are as expected.

Reply:
good catch, done, found another test code bug in doing so :)